tools/fapi/tss2_template.c

```diff
@@ -470,25 +470,25 @@ char* ask_for_password() {
 #else /* FAPI_3_0 */
     char *pw;
 #endif /* FAPI_3_0 */
-    char *password = NULL;
```

Comment:
this is why when I do globals, I make them start with an underscore or put them in a structure with a variable called global or something distinct... ive had these bugs bite me in the A$$